ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/GAnalytics.cpp

```diff
@@ -78,7 +81,12 @@ void GAnalytics::init()
 #endif // CPP_VIEWER / QML_VIEWER
 
   const auto os = QOperatingSystemVersion::current();
-  m_defaultParameters.insert("operating_system", QString{"%1 %2.%3"}.arg(os.name(), os.majorVersion(), os.minorVersion()));
+  m_defaultParameters.insert("operating_system", QString{"%1 %2.%3"}.arg(os.name()).arg(os.majorVersion()).arg(os.minorVersion()));
+
+  m_googleAnalyticsUrl = QString{"https://google-analytics.com/mp/collect?api_secret=%1&measurement_id=%2"}.arg(m_apiSecret).arg(m_measurementId);
```

Comment:
Same comment, use multi-arg `arg` and see if either of these args are numerics. Use QString::number in that case.